workflow/snakemake/Snakefile

```diff
@@ -0,0 +1,44 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ mkdir snakemake-local-run
+#   $ cd snakemake-local-run
+#   $ virtualenv ~/.virtualenvs/root6-roofit-snakemake
+#   $ source ~/.virtualenvs/root6-roofit-snakemake/bin/activate
+#   $ pip install snakemake
+#   $ cp -a ../code ../workflow/snakemake/inputs.yaml .
+#   $ snakemake -s ../workflow/snakemake/Snakefile -p --cores 1
+
+configfile: "inputs.yaml"
+
+rule all:
+    input:
+        "results/data.root",
+        "results/plot.png"
+        
+
+rule gen_data:
```

Comment:
We can just call the rule `gendata` as in the other specifications. Same for `fitdata`.